client/src/client-only-routes/ShowProjectLinks.js

```diff
@@ -0,0 +1,194 @@
+/* eslint-disable react/jsx-sort-props */
+import React, { useState } from 'react';
+import PropTypes from 'prop-types';
+import '../components/layouts/project-links.css';
+import { Button, Modal } from '@freecodecamp/react-bootstrap';
+import { maybeUrlRE } from '../utils';
+import { Spacer } from '../components/helpers';
+import { createSelector } from 'reselect';
+import { projectMap } from '../resources/certAndProjectMap';
+import { Link } from 'gatsby';
+import SolutionViewer from '../components/settings/SolutionViewer';
+import { find } from 'lodash';
+import { userSelector } from '../redux';
```

Comment:
The problem is that this fetches the currently logged in user (i.e. the sessionUser), not the user whose cert you're looking at.

What I recommend is taking a look at `fetch-user-saga.js` and `fetchOtherUser`.  It might be sufficient to use `fetchProfileForUser`, but that might a) be overkill and b) have side-effects.  Still, that's where I'd start.

Also, it might be worth looking at `Profile.js` since that uses `fetchProfileForUser` and is doing similar stuff to this PR.

Reply:
Thanks for that. I had dismissed this method, because I was being silly with the way I was trying to get `maybeUser`.